GitCommands/Git/GitModule.cs

```diff
@@ -1116,19 +1116,11 @@ public bool ExistsMergeCommit(string? startRev, string? endRev)
             {
                 "--parents",
                 "--no-walk",
+                "--min-parents=2",
```

Comment:
This option is available in Git 2.1 at least (but maybe not when the command was originally added?), so no version check required